### PR TITLE
Clean up of recipes folder

### PR DESCRIPTION
**Pull request by dominic-blain**
Removed some useless recipes files.

### CODE REVIEW

File: workspace/utilities/recipes/recipe-custom.xsl [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xsl:stylesheet version="1.0" xmlns:exsl="http://exslt.org/common">
+
+
+</xsl:stylesheet>
```
Comment (nitriques) [inline workspace/utilities/recipes/recipe-custom.xsl]: Please add an empty line at the end of every file

File: workspace/utilities/recipes/recipe-typography.xsl [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xsl:stylesheet version="1.0" xmlns:exsl="http://exslt.org/common">
+
+
+</xsl:stylesheet>
```
Comment (nitriques) [inline workspace/utilities/recipes/recipe-typography.xsl]: Same here

File: workspace/utilities/recipes/recipes.xsl [hunk 3]
```diff
@@ -1,16 +1,7 @@
...
-	<xsl:import href="recipe-colors.xsl" />
-	<xsl:import href="recipe-spacings.xsl" />
+<xsl:import href="recipe-typography.xsl" />
+<xsl:import href="recipe-custom.xsl" />
 
-	<xsl:import href="recipe-layout.xsl" />
-
-	<xsl:import href="recipe-buttons.xsl" />
-	<xsl:import href="recipe-graphics.xsl" />
 </xsl:stylesheet>
```
Comment (nitriques) [inline workspace/utilities/recipes/recipes.xsl]: Same here

**Comment by dominic-blain** [comment]
Ping @nitriques :)

**Pull request merged**